1907-11-07.xml

```diff
@@ -1704,233 +1724,240 @@
-                    <p>Schmid and Co. 193</p>
-                    <p>P. Hahnloser and Co, 371</p>
-                    <p>A. Hess and Co 40</p>
-                    <p>W. Trapp and Co., 63</p>
+                    <p>R. and O. Lindemann, 800 " "</p>
+                    <p>J. Planta and Co., 384 " "</p>
+                    <p>Mohr and Fender 78 " "</p>
+                    <p>G. Frauger and Co., 300 " "</p>
+                    <p>G. Riecken, 281 " "</p>
+                    <p>W. Getty and Co., 210 " "</p>
+                    <p>Moursi Bros., 120 " "</p>
+                    <p>F. Andres, 340 " "</p>
+                    <p>1. Bindernagel, 135 " "</p>
+                    <p>Choremi, Benachi and Co., 255 " "</p>
+                    <p>E. Mallison and Co., 61 " "</p>
+                    <p>Peel and Co., 351 " "</p>
+                    <p>Schmid and Co. 193 " "</p>
+                    <p>P. Hahnloser and Co, 371 " "</p>
+                    <p>A. Hess and Co 40 " "</p>
+                    <p>W. Trapp and Co., 63 " "</p>
                     <p>4188 bales cotton</p>
                     <p>A. Seeger and Co., 24 barrels and cases guts</p>
                     <p>Klink and Lauer, 16 barrels fresh fish</p>
                     <p>Kyriasi Bros., 4 cases cigarettes</p>
                     <p>Various, 19 packages sundries</p>
                     <p>For DUNKIRK, by the s.8. Cyprian Prince, sailed on the 2nd November :</p>
                     <p>Carver Bros. and Co, Ltd, 710 bales cotton</p>
-                    <p>G. Frauger and Co., 734</p>
-                    <p>Peel and Co., 640</p>
-                    <p>H. Bindernagel, 325</p>
-                    <p>P. Hahnloser and Co., 211.</p>
-                    <p>Choremi, Benschi and Co., 110</p>
-                    <p>R. and O. Lindemann, 75</p>
-                    <p>E. Mallison and Co., 65</p>
-                    <p>Schmid and Co., 31</p>
-                    <p>Mohr and Fenderl, 93</p>
-                    <p>G. Riecken, 30</p>
-                    <p>F. Andres, 15</p>
-                    <p>J. Planta and Co., 220</p>
+                    <p>G. Frauger and Co., 734 " "</p>
+                    <p>Peel and Co., 640 " "</p>
+                    <p>H. Bindernagel, 325 " "</p>
+                    <p>P. Hahnloser and Co., 211 " "</p>
+                    <p>Choremi, Benschi and Co., 110 " "</p>
+                    <p>R. and O. Lindemann, 75 " "</p>
+                    <p>E. Mallison and Co., 65 " "</p>
+                    <p>Schmid and Co., 31 " "</p>
+                    <p>Mohr and Fenderl, 93 " "</p>
+                    <p>G. Riecken, 30 " "</p>
+                    <p>F. Andres, 15 " "</p>
+                    <p>J. Planta and Co., 220 " "</p>
                     <p>3,259 bales cotton</p>
                     <p>Behrend and Co, 537 tons cotton seed</p>
                 </div>
-                <div type="item">
+                <div type="item" status="verified">
```

Comment:
use the template: https://dig-eg-gaz.github.io/issues/templates/#telegramme-havas (it may need some adaptation)